components/canvas/webgl_thread.rs

```diff
@@ -276,6 +287,11 @@ impl WebGLThread {
                             }
                         }
 
+                        if is_pathfinder_context {
+                            let framebuffer = data.ctx.framebuffer();
+                            let gl_device = GLDevice::new(GLVersion::GLES3,framebuffer);
```

Comment:
@jdm 

So, the next one to set up is the Pathfinder renderer, but what do I do for the resource loader within it?

Reply:
I think you can probably use https://github.com/servo/pathfinder/blob/652a03eb2b90eef7de78cd31331decf12036a960/demo/magicleap/src/magicleap.rs#L182 and copy the contents of https://github.com/servo/pathfinder/tree/master/resources into https://github.com/servo/servo/tree/master/resources.